Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -1869,6 +1881,10 @@ - (void)sectionEditorFinishedEditing:(SectionEditorViewController *)sectionEdito
     self.skipFetchOnViewDidAppear = YES;
     [self dismissViewControllerAnimated:YES completion:NULL];
     if (didChange) {
+        __weak typeof(self) weakSelf = self;
+        self.articleContentLoadCompletion = ^{
+            [weakSelf.webViewController scrollToSection:sectionEditorViewController.section animated:YES];
```

Comment:
This is OK without the corresponding strongify because the block above that asyncs to the main queue strongly captures self. If it were a weak self above, self could be deallocated between when the async call is made an the block is executed